dds/idl/be_produce.cpp

```diff
@@ -138,18 +138,14 @@ string to_macro(const char* fn)
 #ifdef max
 #undef max
 #endif
```

Comment:
I think you can get rid of these as well, unless `max` is being used somewhere below.

Reply:
Done